### ISSUE TITLE
undefined reference to `std::__cxx11::basic_ostringstream

### ISSUE THREAD

**Issue by favaleandro** [ki9gpin/WinBGIm-64#2]
Hi, 

Sorry, I am a begginer on graphics programing using C/C++.

When compiling a little CPP test program using the function outtext() ...

#include <stdio.h>
#include <graphics.h>
int main() {
    initwindow(600, 600, "Test Window");
    char q[] = "Text";
    outtext(q);
    getch();
    return 0;
}

I get the following error:

F:\Code\CPP\Graphics\winbgim\WinBGIm-64\libbgi\src\text.cxx|29|undefined reference to `std::__cxx11::basic_ostringstream<char, std::char_traits<char>, std::allocator<char> >::basic_ostringstream()'|

I am using Code Blocks with MinGW (x86_64-w64-mingw32).

**Comment by ki9gpin** [comment]
So you configured codeblocks to use both libbgi.a and graphics.h  ?

**Comment by ki9gpin** [comment]
Try setting  gcc g++ version in  compiler seetings to c++ 11 or above . Library is compiled with c++11.

**Comment by favaleandro** [comment]
> So you configured codeblocks to use both libbgi.a and graphics.h ?

Yes. When using only drawing functions, like circle(), drawpoly(), line(), everything goes well.
Thank you.

**Comment by ki9gpin** [comment]
It's possible that mingw in codeblocks is using c++03 or older. Change that to c++11. 
https://stackoverflow.com/a/24398366

Look here if you want to know why. 
https://github.com/preshing/junction/issues/37#issuecomment-455702630

**Comment by favaleandro** [comment]
> It's possible that mingw in codeblocks is using c++03 or older. Change that to c++11. https://stackoverflow.com/a/24398366
> 
> Look here if you want to know why. [preshing/junction#37 (comment)](https://github.com/preshing/junction/issues/37#issuecomment-455702630)

Thank you for the fast response.
I tried this, but the error is still raised.

**Comment by ki9gpin** [comment]
You can clone this repo and  load as a project in Visual Studio Code.
